src/client/components/Navigation/Topnav.less

```diff
@@ -86,6 +86,18 @@
     }
   }
 
+  &__popover {
+    & li {
+      display: block;
+
+      @media @antd-medium {
+        &:nth-child(2) {
```

Comment:
I don't think this is good practice to display: none on the nth-child, needs to be more explicit imo

Reply:
I agree @jm90m . But to do that I would have to give that particular list item a class. I just made a commit to do just that.